foundryContract/src/libraries/Orakl.sol

```diff
@@ -0,0 +1,121 @@
+// SPDX-License-Identifier: MIT
```

Comment:
I don't think we need this file. It is used for Request-Response. Please remove it.